tests/functional/u/using_constant_test.txt

```diff
@@ -26,3 +26,5 @@ using-constant-test:89:3:89:15::Using a conditional statement with a constant va
 using-constant-test:93:3:93:18::Using a conditional statement with a constant value:INFERENCE
 comparison-of-constants:117:3:117:8::"Comparison between constants: '2 < 3' has a constant value":HIGH
 using-constant-test:156:0:157:8::Using a conditional statement with a constant value:INFERENCE
+using-constant-test:168:3:168:4::Using a conditional statement with a constant value:UNDEFINED
```

Comment:
This message has confidence "undefined", it's a little strange but we can merge anyway, I'm just pointing it out in case you did not notice.